### PR TITLE
Prepare Zoe and Cosmic-Swingset for the release of Zoe v0.1.0

### PR DESCRIPTION
**Pull request by katelynsills**
This PR prepares Zoe and Cosmic-Swingset for the release of Zoe v0.1.0. 

Following this, I will:
1. Add the tag `zoe-v0.1.0` to master.
2. Do `yarn publish` for `@agoric/zoe`
3. Do `yarn install @agoric/zoe` in Cosmic-Swinget (only tested through `yarn link` so far)
4. Confirm that our "Getting Started" documentation steps still work (making changes as necessary)

### CODE REVIEW

File: packages/zoe/zoe.js [hunk 1]
```diff
@@ -227,6 +227,7 @@ const makeZoe = async (additionalEndowments = {}) => {
   // securely escrow and get an escrow receipt and payouts in return.
 
   const zoeService = harden({
+    getVersion: () => 'v0.1.0',
```
Comment (michaelfig) [inline packages/zoe/zoe.js:230]: Is there some way to avoid hardcoding this?

Probably not.

File: packages/zoe/package.json [hunk 2]
```diff
@@ -47,5 +47,17 @@
     "tap-spec": "^5.0.0",
     "tape": "^4.11.0",
     "tape-promise": "^4.0.0"
-  }
+  },
+  "files": [
```
Comment (michaelfig) [inline packages/zoe/package.json:51]: I would prefer not specifying files, and instead using .npmignore.

YMMV.

**Review by michaelfig — APPROVED** [review]
LGTM!

The comments I made are too minor to require a change before you merge.

File: packages/zoe/zoe.js (hunk 1 above)
Comment (katelynsills) [reply packages/zoe/zoe.js:230]: I looked into some different ways to import package.json, but it seemed to introduce security issues.  There's also `process.env.npm_package_version` but that wasn't working for me.

File: packages/zoe/package.json (hunk 2 above)
Comment (katelynsills) [reply packages/zoe/package.json:51]: Hmm, if it helps, future changes to Zoe should drastically reduce the number of files listed here. I generally prefer the whitelist approach myself.

File: packages/zoe/zoe.js (hunk 1 above)
Comment (warner) [reply packages/zoe/zoe.js:230]: Hm, this seems to be a problem in all languages. In Python I built a thing called [Versioneer](https://github.com/warner/python-versioneer) to grab a version number from the git environment as you do a build.. maybe someone's built something similar for node. Probably as a command that gets run at `npm pack` or `npm build` time, which modifies a file and inserts the current version data just before it gets bundled into the tarball.

**Pull request merged**